### ISSUE TITLE
Record the API endpoint and instance URL in the post metadata

### ISSUE THREAD

**Issue by nikclayton** [dangom/writefreely.el#15]
Since the user can change the API endpoint and instance URL, it's not sufficient (I think) to record the post ID in the metadata about the post,

The user can post to one instance, and then update the instance URL / API endpoint -- when they do that, anything that operates on historical posts (update, delete, get the link, etc) will fail.

**Comment by dangom** [comment]
True. It does make sense to store the endpoint info in the org file (or as a property) as well. I'm somewhat busy with my work right now, but I should be able to tackle this when things calm down a bit.

**Comment by sch1z00** [comment]
my instance doesn't has an api directory, so I've tryed to set "Instance Api Endpoint" with write.as and "Instance Url" with my instance url, but when I publish my posts are on write.as; is it possible this kind of customization?

**Comment by dangom** [comment]
I don't think that is possible. The API is just where we send our requests to. The URL is only used to open and visualize the posts. Unless you provide an API for us to speak to, we cannot manage your posts programmatically.
